CHANGELOG.md

```diff
@@ -293,6 +293,25 @@ If you were using `$govuk-font-family-gds-transport` to set the font on an eleme
 
 This change was introduced in [pull request #3949: Simplify font family settings](https://github.com/alphagov/govuk-frontend/pull/3949).
 
+#### Removal of Javascript polyfills for IE8-11
+
+We've removed polyfills for:
+
+- `DOMTokenList`, `Element.prototype.classList`, `Element.prototype.closest`,`Element.prototype.matches`, and `Event`, required for IE 11 and below
+- `Element.prototype.dataset`, required for IE 10 and below
+- `Date.now`, `Document`, `Element`, `Element.prototype.nextElementSibling`, `Element.prototype.previousElementSibling`, `Function.prototype.bind`, `Object.defineProperty`, `String.prototype.trim` and `Window`, required for IE8
+
+GOV.UK Frontend no longer needs these polyfills because Internet Explorer will no longer try to run our JavaScript, as long as you are including it in your service according to our instructions.
```

Comment:
I might be overly worried, but feels like we should call this out

```suggestion
GOV.UK Frontend no longer needs these polyfills because Internet Explorer will no longer try to run our JavaScript, as long as you are including it in your service according to our instructions using `<script type="module">`.
```